spec/router_spec.js

```diff
@@ -142,6 +143,68 @@ describe("Router", function() {
     });
   });
 
+  describe("navigate", function() {
+    class MyRouter extends Router {
+    }
+
+    class MyView extends View {
+      render() {
+        this.$el.html('');
+        return this;
+      }
+    }
+
+    beforeEach(function() {
+      this.router = new MyRouter({ $el: $('<div>') });
+    });
+
+    it("should call router.navigate right away if view does not define a beforeNavigate function", function() {
+      spyOn(Backbone.history, 'navigate');
```

Comment:
The implementation calls `super.navigate`. Spying on something known to be part of the implementation of `super.navigate` is brittle because the implementation of `super.navigate` could change, breaking these tests.

Can you change this to use a spy on `super.navigate`?